spec/expressions.md

```diff
@@ -663,48 +710,52 @@ F(10, 20, new object\[\] {});\
 F(10, 20, new object\[\] {30, 40});\
 F(10, 20, new object\[\] {1, "hello", 3.0});
 
-In particular, note that an empty array is created when there are zero arguments given for the parameter array. *end example*\]
+In particular, note that an empty array is created when there are zero arguments given for the parameter array. 
+
+When arguments are omitted from a function member with corresponding optional parameters, the default arguments of the function member declaration are implicitly passed. 
 
-When arguments are omitted from a function member with corresponding optional parameters, the default arguments of the function member declaration are implicitly passed. \[*Note*: Because these are always constant, their evaluation will not impact the evaluation of the remaining arguments. *end note*\]
+> [!NOTE]
+> Because these are always constant, their evaluation will not impact the evaluation of the remaining arguments.
 
 ### Type inference
 
 #### General
 
-When a generic method is called without specifying type arguments, a ***type inference*** process attempts to infer type arguments for the call. The presence of type inference allows a more convenient syntax to be used for calling a generic method, and allows the programmer to avoid specifying redundant type information. \[*Example*: Given the method declaration:
+When a generic method is called without specifying type arguments, a ***type inference*** process attempts to infer type arguments for the call. The presence of type inference allows a more convenient syntax to be used for calling a generic method, and allows the programmer to avoid specifying redundant type information. Given the method declaration:
```

Comment:
The "example" marker should not be removed:

```suggestion
When a generic method is called without specifying type arguments, a ***type inference*** process attempts to infer type arguments for the call. The presence of type inference allows a more convenient syntax to be used for calling a generic method, and allows the programmer to avoid specifying redundant type information. [*Example*: Given the method declaration:
```